### PR TITLE
(SIMP-4941) Cleanup unneeded fixtures

### PR DESCRIPTION
**Pull request by jeefberkey**
SIMP-4941 #comment pupmod fixed

### CODE REVIEW

**Comment by jeefberkey** [comment]
https://gitlab.com/simp/pupmod-simp-pupmod/pipelines/23893600

File: .fixtures.yml [hunk 1]
```diff
@@ -1,40 +1,41 @@
...
     puppet_authorization:
       repo: https://github.com/simp/puppetlabs-puppet_authorization
       ref: 0.4.0
+
+    # Fixtures needed for the acceptance test
+    # apache: https://github.com/simp/pupmod-simp-apache
+    augeasproviders_ssh: https://github.com/simp/augeasproviders_ssh
+    # logrotate: https://github.com/simp/pupmod-simp-logrotate
+    # oddjob: https://github.com/simp/pupmod-simp-oddjob
+    # pki: https://github.com/simp/pupmod-simp-pki
```
Comment (trevor-vaughan) [inline .fixtures.yml]: Why are these all commented out?

Comment (jeefberkey) [reply .fixtures.yml]: This one was still WIP when I went out of office. I was unsure it was going to pass, but it did

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,6 @@
+* Thu Jun 14 2018 Nick Miller <nick.miller@onyxpoint.com> - 7.6.0-0
+- Update systemd fixtures and CI assets
```
Comment (trevor-vaughan) [inline CHANGELOG]: This doesn't appear to update anything related to `systemd`, is something missing or is this message incorrect?

**Comment by jeefberkey** [comment]
https://gitlab.com/simp/pupmod-simp-pupmod/pipelines/24533433

**Pull request merged**